packages/desktop-client/src/components/manager/BudgetFileSelection.tsx

```diff
@@ -202,7 +204,7 @@ function BudgetFileState({ file, currentUserId }: BudgetFileStateProps) {
       break;
     case 'local':
       Icon = SvgFileDouble;
-      status = 'Local';
+      status = t('Local');
       ownerName = 'You';
```

Comment:
Should we translate this? (Probably with some context)

Reply:
I think we tried context at one point and it displayed weirdly and we had to revert.

Reply:
Makes sense. We should probably translate though to match the other `You`'s in this file

Reply:
Oh, sorry, I saw it as a comment on the line above...

Reply:
31f6f0d0c60190b74aa571ffad2a063249c5145c